package.json

```diff
@@ -5,10 +5,11 @@
   "main": "index.js",
   "scripts": {
     "build": "lerna exec --parallel -- babel src -d dist --ignore '**/*.test.jsx,**/*.stories.jsx' --root-mode upward",
-    "ci:packages": "npm ci && lerna bootstrap --ci --hoist --force-local",
+    "ci:packages": "npm ci && lerna clean --yes && lerna bootstrap --ci --hoist --nohoist=@bbc/*",
```

Comment:
Instead of having `--hoist --nohoist=@bbc/*` could we explicitly just hoist styled-components?
e.g. `--hoist=styled-components`?

Reply:
I have tried doing this, however lerna seems flakey when asking to hoist a specific package. It always misses a SC instance. Tried many different hoisting options with no success

Reply:
Thanks for looking into this, Drew.